index.js

```diff
@@ -17,6 +18,10 @@ module.exports = (options = {}) => {
 	directory = pkgDir.sync(directory);
 
 	if (directory) {
+		if (!isWritable.sync(path.join(directory, 'node_modules'))) {
```

Comment:
We don't need a module for this. Just add a helper method:

```js
const isWritable = path => {
	try {
		fs.accessSync(path, fs.constants.W_OK);
		return true
	} catch (_) {
		return false
	}
};
```

Reply:
Refactored.